cmd/healthbot/main.go

```diff
@@ -56,3 +72,28 @@ func main() {
 	wg.Wait()
 	log.Info().Msg("daemon closed")
 }
+
+func getChain(chain string) (client.Chain, error) {
+	switch chain {
+	case "ethereum":
+		return client.Chains.Ethereum, nil
+	case "optimism":
+		return client.Chains.Optimism, nil
+	case "polygon":
+		return client.Chains.Polygon, nil
+	case "ethereum-goerli":
+		return client.Chains.EthereumGoerli, nil
+	case "optimism-kovan":
+		return client.Chains.OptimismKovan, nil
+	case "optimism-goerli":
+		return client.Chains.OptimismGoerli, nil
+	case "arbitrum-goerli":
+		return client.Chains.ArbitrumGoerli, nil
+	case "polygon-mumbai":
+		return client.Chains.PolygonMumbai, nil
+	case "local":
+		return client.Chains.Local, nil
+	default:
+		return client.Chain{}, fmt.Errorf("%s is not a valid chain", chain)
+	}
+}
```

Comment:
If we go this route, the chain names used in the config need to be consistent with the names here. These names are what is used in the JS SDK/CLI as well as the Rigs CLI.